MCNPInput.cpp

```diff
@@ -54,6 +54,7 @@ static int makeint( const std::string& token ){
   char* end;
   int ret = strtol(str, &end, 10);
   if( end != str+token.length() ){
+    record << "Warning: string [" << token << "] did not convert to int as expected." << std::endl;
     std::cerr << "Warning: string [" << token << "] did not convert to int as expected." << std::endl;
```

Comment:
Don't bother -- if you're planning to clean it up in in a future PR that's fine.

(Output and logging in c++ is always a colorful bikeshed.  In the past I've written custom ostream subclasses that can optionally direct to zero, one, or multiple outputs (stdout, file, etc.), but it's a hassle to maintain.  Lavtely I've found it most comfortable to create macros that accept ostream syntax.  This provides a nice way to centrally control verbosity, write to multiple outputs, compile out DEBUG statements, etc.)